misc/toml-parser/src/main/resources/syntax_tree_descriptor.json

```diff
@@ -1,191 +1,211 @@
-        },
-        {
-          "name": "fields",
-          "type": "KeyValueNode",
-          "occurrences": "MULTIPLE"
-        }
-      ]
-    },
-    {
-      "name": "TableArrayNode",
-      "base": "DocumentMemberDeclarationNode",
-      "kind": "TABLE_ARRAY",
-      "attributes": [
-        {
-          "name": "firstOpenBracket",
-          "type": "Token"
+            "name": "DocumentMemberDeclarationNode",
+            "base": "Node",
+            "isAbstract": true
         },
         {
-          "name": "secondOpenBracket",
-          "type": "Token"
+            "name": "ValueNode",
+            "base": "Node",
+            "isAbstract": true
         },
         {
-          "name": "identifier",
-          "type": "ValueNode",
-          "occurrences": "MULTIPLE_SEPARATED"
+            "name": "TableNode",
+            "base": "DocumentMemberDeclarationNode",
+            "kind": "TABLE",
+            "attributes": [
+                {
+                    "name": "openBracket",
+                    "type": "Token"
+                },
+                {
+                    "name": "identifier",
+                    "type": "ValueNode",
+                    "occurrences": "MULTIPLE_SEPARATED"
+                },
+                {
+                    "name": "closeBracket",
+                    "type": "Token"
+                },
+                {
+                    "name": "fields",
+                    "type": "KeyValueNode",
+                    "occurrences": "MULTIPLE"
+                },
+                {
+                    "name": "newLines",
+                    "type": "Token",
+                    "occurrences": "MULTIPLE"
+                }
+            ]
         },
         {
-          "name": "firstCloseBracket",
-          "type": "Token"
+            "name": "TableArrayNode",
+            "base": "DocumentMemberDeclarationNode",
+            "kind": "TABLE_ARRAY",
+            "attributes": [
+                {
+                    "name": "firstOpenBracket",
+                    "type": "Token"
+                },
+                {
+                    "name": "secondOpenBracket",
+                    "type": "Token"
+                },
+                {
+                    "name": "identifier",
+                    "type": "ValueNode",
+                    "occurrences": "MULTIPLE_SEPARATED"
+                },
+                {
+                    "name": "firstCloseBracket",
+                    "type": "Token"
+                },
+                {
+                    "name": "secondCloseBracket",
+                    "type": "Token"
+                },
+                {
+                    "name": "fields",
+                    "type": "KeyValueNode",
+                    "occurrences": "MULTIPLE"
+                },
+                {
+                    "name": "newLines",
+                    "type": "Token",
+                    "occurrences": "MULTIPLE"
+                }
```

Comment:
Newlines should be part of minutiae

Reply:
Let's discuss here,
https://github.com/ballerina-platform/ballerina-lang/pull/26429#issuecomment-720410806